package/crossplane.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: meta.pkg.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: provider-template
+  annotations:
+    company: Template
+    maintainer: template <template@mail.com>
+    source: github.com/crossplane/provider-template
+    license: Apache-2.0
+
+spec:
+  controller:
+    image: crossplane/provider-template-controller:v0.0.1
```

nitpick:
```suggestion
    image: crossplane/provider-template-controller:latest
```
@salaboy maybe best to match the `Makefile` for now?